dockerfile.template

```diff
@@ -15,21 +15,23 @@ RUN apt-get -qq install -y --no-install-recommends \
 RUN wget -q https://bootstrap.pypa.io/get-pip.py
 RUN python2.7 get-pip.py
 
-RUN pip2 -q install --upgrade pip
-RUN pip2 -q install setuptools
-RUN pip2 -q install azure azure-cli
+RUN python2.7 -m pip -q install pip==9.0.3
+RUN python2.7 -m pip -q install --upgrade setuptools
+RUN python2.7 -m pip -q install --upgrade azure azure-cli
 
 # Install Python3 and Python 3 dependencies
 RUN rm -rf ~/.cache/pip
 RUN apt-get -qq install -y --no-install-recommends \
                 python3 \
                 python3-pip
 
-RUN pip3 -q install --upgrade pip
-RUN pip3 -q install setuptools
+RUN python3 -m pip -q install --upgrade pip
+RUN python3 -m pip -q install --upgrade setuptools
+RUN python3 -m pip -q install pyyaml~=4.2b4
+RUN python3 -m pip -q install azure-keyvault==0.3.7 
 
 #task.py in aether uses appliction insights for reporting. 
-RUN pip3 -q install applicationinsights==0.11.3
+RUN python3 -m pip -q install applicationinsights==0.11.3
```

Comment:
this is already in the msftkube's whl's requreiments/setup.py why do we need to call it out explicitly?

Reply:
I believe you added this, but the reason is because this isn't the repo with that requirements.txt file is in so when we install this stuff from the wheel we end up with mismatching versions.

Reply:
I am also working on getting this set up properly via a docker container created from msftkube builds directly